test/framework/ip/incluster.go

```diff
@@ -39,7 +39,7 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/klog/v2"
 	"k8s.io/utils/ptr"
-	ipamv1 "sigs.k8s.io/cluster-api/exp/ipam/api/v1alpha1"
+	ipamv1 "sigs.k8s.io/cluster-api/exp/ipam/api/v1beta1"
```

Comment:
Maybe we need to keep the old api group for now for the test code, because in CI v1beta1 is not yet deployed (it is only used for IPAM and thus may be not up to date.

Reply:
Yes, makes sense. I was just looking at why the tests were failing and how the testing framework is constructed. Thanks!